1448348365343	Sync.ErrorHandler	DEBUG	Flushing file log.
1448348365344	Sync.Status	DEBUG	Status.sync: success.sync => error.sync.prolonged_failure
1448348365344	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1448348365346	Sync.BrowserIDManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})
1448348365350	Sync.BrowserIDManager	INFO	currentAuthState returning error.login.reason.network due to previous failure
1448348365350	Sync.Status	DEBUG	Status.login: error.login.reason.network => error.login.reason.network
1448348365350	Sync.Status	DEBUG	Status.service: error.sync.failed => error.login.failed
1448348365353	Sync.ErrorHandler	DEBUG	Log cleanup threshold time: 1447484365353
1448348365353	Sync.ErrorHandler	DEBUG	No logs to clean up.
1448348380201	Sync.Resource	DEBUG	mesg: GET fail 401 https://sync-220-us-west-2.sync.services.mozilla.com/1.5/23370112/info/collections
1448348380202	Sync.Resource	DEBUG	GET fail 401 https://sync-220-us-west-2.sync.services.mozilla.com/1.5/23370112/info/collections
1448348380202	Sync.Service	INFO	Logging out
1448348380202	Sync.BrowserIDManager	INFO	currentAuthState returning error.login.reason.network due to previous failure
1448348380202	Sync.Status	DEBUG	Status.login: error.login.reason.network => error.login.reason.network
1448348380202	Sync.Status	DEBUG	Status.service: error.login.failed => error.login.failed
1448348380203	Sync.ErrorHandler	INFO	Got 401 response; resetting clusterURL.
1448348380203	Sync.ErrorHandler	DEBUG	New mid-sync 401 failure. Making a note.
1448348380204	Sync.ErrorHandler	INFO	Attempting to schedule another sync.
1448348380205	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms.
